bridge-sdk/bridge-clients/near-bridge-client/src/near_bridge_client.rs

```diff
@@ -4,8 +4,10 @@ use bridge_connector_common::result::{BridgeSdkError, Result};
 use derive_builder::Builder;
 use near_contract_standards::storage_management::StorageBalance;
 use near_crypto::{SecretKey, Signer};
+use near_primitives::types::Gas;
 use near_primitives::{hash::CryptoHash, types::AccountId, views::TxExecutionStatus};
 use near_rpc_client::{ChangeRequest, ViewRequest};
+use near_sdk::json_types::U128;
```

Comment:
We avoided using `near_sdk`. Can we get U128 from `near_primitives` or a different crate?

Reply:
fixed: https://github.com/Near-One/bridge-sdk-rs/pull/139/commits/dffcbdcd2bf2b5b9afae84aa6a33dd08f3eb2278

Reply:
Why aren't we using near_sdk? I checked, and it's implicitly included in our dependencies. And I didn't just add it now. For example, omni-types or near-contract-standards depend on it. Should we create a separate issue to remove near_sdk from the dependencies?

Reply:
Just generally didn't want to use the whole of `near_sdk` just for the types. It's true that it's currently used in `omni-types`. I think we had an idea to hide it under a feature flag. But maybe it's not very important right now